web/modules/weather_data/src/Service/Test/LegacyIconMapping.php.test

```diff
@@ -18,9 +18,7 @@ use PHPUnit\Framework\TestCase;
  */
 final class WeatherDataLegacyIconMapping extends TestCase
 {
-    /**
-     * @group unit
-     */
+    #[Group(unit)]
```

Comment:
Was trying to find docs about this syntax and couldn't, but if it's all good please ignore

Reply:
Oh oops, nope, that needs to come back out. I got off course trying to use a newer version of phpunit and forgot to unwind it all.